src/lsm/forest_fuzz.zig

```diff
@@ -370,8 +499,11 @@ pub fn generate_fuzz_ops(random: std.rand.Random, fuzz_op_count: usize) ![]const
                     // Can only checkpoint on the last beat of the bar.
                     compact_op % constants.lsm_batch_multiple == constants.lsm_batch_multiple - 1 and
                     compact_op > constants.lsm_batch_multiple and
-                    // Checkpoint at roughly the same rate as log wraparound.
-                    random.uintLessThan(usize, Environment.compacts_per_checkpoint) == 0;
+                    // Never checkpoint at the same op twice
+                    compact_op > checkpointed_op + constants.lsm_batch_multiple;
```

Comment:
This now checkpoints at the end of every measure. The replica checkpoints (slightly more than) every `journal_slot_count / lsm_batch_multiple == 16` measures. That's what the `compacts_per_checkpoint` was for.

Reply:
Yeah, that was what I intended, but now I understand that that was the wrong thing to do.